minorminer/utils/zephyr/coordinate_systems.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2025 D-Wave
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# ================================================================================================
+
+
+from __future__ import annotations
+
+from collections import namedtuple
+
```

Comment:
Would add an `__all__` to this file as well as `node_edge.py` and `plane_shift.py`.
```python
__all__ = ["cartesian_to_zephyr", "zephyr_to_cartesian"]
```
Not sure if `zephyr_fields`, `ZephyrCoord`, etc. below should be there as well.